src/rng.rs

```diff
@@ -15,8 +15,8 @@ use crate::time::Hertz;
 
 #[derive(Debug)]
 pub enum ErrorKind {
-    ClockError,
-    SeedError,
+    ClockError = 1,
+    SeedError = 2,
```

Comment:
I think technically this is a breaking change, so should go in the CHANGELOG

Reply:
Better to keep breaking changes to a minimum. The only reason I did this was because I wanted to be explicit about what `ErrorEnum as u32` would result in. Changing this to 
```
pub enum ErrorKind {
    ClockError = 0,
    SeedError = 1,
}
```
wouldn't be a breaking change but still being explicit.

Reply:
Ok, great!